packages/react-dom/src/server/escapeScriptForBrowser.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import {checkHtmlStringCoercion} from 'shared/CheckStringCoercion';
+
+const scriptRegex = /(<\/|<)(s)(cript)/gi;
+const scriptReplacer = (match, prefix, s, suffix) =>
+  `${prefix}${substitutions[s]}${suffix}`;
+const substitutions = {
+  s: '\\u0073',
+  S: '\\u0053',
+};
+
+/**
+ * Escapes javascript for embedding into HTML.
+ *
+ * @param {*} scriptText Text value to escape.
+ * @return {string} An escaped string.
+ */
+function escapeScriptForBrowser(scriptText) {
```

Comment:
Let's inline this where it's used because this strategy hasn't been vetted for a general usage. It's only valid for that exact use case but this makes it seems like it's legit to reuse this function.

Reply:
done